<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-2ACJK3Q" failures="1" tests="1" name="Default test(failed)" time="6.452" errors="0" timestamp="2025-10-14T12:14:14 IST">
  <testcase classname="practise.test.invoiceTest" name="createinvoiceTest" time="1.839">
    <failure type="java.lang.AssertionError" message="expected [login] but found [Admin123@ Administrator1 - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [login] but found [Admin123@ Administrator1 - Home - vtiger CRM 5 - Commercial Open Source CRM]
at practise.test.invoiceTest.createinvoiceTest(invoiceTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createinvoiceTest -->
  <testcase classname="practise.test.invoiceTest" name="createinvoicewithcontactTest" time="0.0">
    <ignored/>
  </testcase> <!-- createinvoicewithcontactTest -->
</testsuite> <!-- Default test(failed) -->
